sourced/ml/core/algorithms/token_parser.py

```diff
@@ -76,6 +80,23 @@ def max_token_length(self, value):
             raise ValueError("max_token_length must be greater than 0 - got %d" % value)
         self._max_token_length = value
 
+    @property
+    def use_nn(self):
+        return self._use_nn
+
+    @use_nn.setter
+    def use_nn(self, value):
+        self._init_nn(value)
+
+    def _init_nn(self, value):
+        if value:
+            self._use_nn = True
+            from sourced.ml.core.models.id_splitter import IdentifierSplitterBiLSTM
+            import modelforge.backends
+            backend = modelforge.backends.create_backend()
```

Comment:
`backend` is not used below, so I guess can be removed

Reply:
Done